### ISSUE TITLE
Adding an entity instead of array of components to engine?

### ISSUE THREAD

**Issue by martikaljuve** [BlackDice/scent#8]
It would be nice to have `engine.addEntity(entity)` in addition to `engine.addEntity(components)`. This would be useful for creating parent-child relationships without having to add the parents to the engine first, e.g.

``` js
// ./components.js
import { Component } from 'scent';

export const cParent = new Component('parent', 'entityRef');
```

``` js
// ./prefabs/some_child_entity.js
import { Entity } from 'scent';
import { cParent } from '../components';

export default function SomeChildEntity(parent) {
    var entity = new Entity();

    entity.add(new cParent([parent]));
    /../

    return entity;
}
```

``` js
// ./index.js
import { Engine, Entity } from 'scent';
import SomeChildEntity from './prefabs/some_child_entity';

var eParent = new Entity([...]);
var eChild = new SomeChildEntity(eParent);

var engine = new Engine();

engine.addEntity(eParent);
engine.addEntity(eChild);
```

Right now I could achieve the same using this:

``` js
// ./index.js

/../

var engine = new Engine();

var eParent = engine.addEntity([...]);
var parent = new cParent([eParent]);
var eChild = engine.addEntity([parent, ...]);
```

**Comment by danielkcz** [comment]
Hey. I am kinda busy now. Feel free to create PR as it sounds good for sure.

**Comment by danielkcz** [comment]
Closed by #9
